packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -99,16 +99,16 @@ export function getBeaconPoolApi({
             // when a validator is configured with multiple beacon node urls, this attestation data may come from another beacon node
             // and the block hasn't been in our forkchoice since we haven't seen / processing that block
             // see https://github.com/ChainSafe/lodestar/issues/5098
-            const {indexedAttestation, subnet, attDataRootHex, committeeIndex} = await validateGossipFnRetryUnknownRoot(
-              validateFn,
-              network,
-              chain,
-              slot,
-              beaconBlockRoot
-            );
+            const {indexedAttestation, subnet, attDataRootHex, committeeIndex, participationIndex} =
+              await validateGossipFnRetryUnknownRoot(validateFn, network, chain, slot, beaconBlockRoot);
 
             if (network.shouldAggregate(subnet, slot)) {
-              const insertOutcome = chain.attestationPool.add(committeeIndex, attestation, attDataRootHex);
+              const insertOutcome = chain.attestationPool.add(
+                committeeIndex,
+                participationIndex,
```

Comment:
what about calling this `validatorCommitteeIndex`?

Reply:
imo `bitIndex` or current name is good

Reply:
will change to `committeeValidatorIndex` as in devnet 5 https://github.com/ChainSafe/lodestar/pull/7265/files#diff-c702dde1ffa90a6d2720c7b4930bc5c08cc1f302cfd09ed9e74d80755b92da8fR645